cwl_tes/main.py

```
@@ -103,6 +103,15 @@ def ftp_upload(base_url, fs_access, cwl_obj):
                 fs_access.upload(source, cwl_obj["location"])
 
 
+def load_public_key(kid):
+    try:
+        with open(kid, 'r') as f:
+            return f.read()
+    except IOError:
+        print("File " + kid + " not found. Using --token-public-key value.")
```

We could log these and similar errors rather than printing (I always forget to do this!). But if the rest of the codebase also prints error messages I wouldn't bother changing this

---

The rest of the code actually raises errors rather than printing or logging, so I am doing the same here now.